src/containers/WithQAing.jsx

```diff
@@ -105,6 +106,13 @@ export const WithQAing = ComposedComponent => {
       return `${numOfGalaxies} galaxies & ${numOfSupernovae} supernovae`;
     }
 
+    getGalaxiesContent(data) {
+      const numGalaxies = Object.keys(data).length;
+      return `${numGalaxies} ${
+        numGalaxies > 1 ? 'galaxies' : 'galaxy'
```

Comment:
Maybe also do `numGalaxies > 1 || numGalaxies === 0 ? 'galaxies' : 'galaxy'`
Not sure if we'll ever have 0 galaxies, but at least if we did the text would show correctly?

Reply:
Good catch, but in this case content is not rendered unless there is an answer present (a selection), and you are not able to "select" 0 things so we can confidently assume numGalaxies >= 1

Reply:
Awesome! That's what I figured, but just wanted to be certain.